engine/base.ftl

```diff
@@ -71,11 +71,20 @@
 [/#function]
 
 [#function intSquareRoot value ]
+    [#local lastPerfect = 1 ]
     [#list 1..100 as square]
```

Comment:
It looks like break directives in lists are [deprecated](https://freemarker.apache.org/docs/ref_directive_list.html#ref.directive.list.break). Should we try and avoid that? It recommends using `?take_while()` instead. Something like:

```
[#list (1..100)?take_while(s -> ((s * s) < value)) as square]
```

Reply:
Couldn't get this to work, still seems to be missing the value when it goes to the next one up.

Reply:
Couldn't get this to work, still seems to be missing the value when it goes to the next one up.

Reply:
Not really an issue then, just something I picked out as I hadn't seen a `break` used here before.